a3p-integration/proposals/p:upgrade-19/swapAnchorForMintedSeat/swap-anchor-for-minted-seat.js

```diff
@@ -0,0 +1,54 @@
+// @ts-nocheck
+/* eslint-disable no-undef */
+
+const swapAnchorForMintedSeat = async powers => {
+  const {
+    consume: { zoe, contractKits: contractKitsP, psmKit: psmKitP, agoricNames },
+  } = powers;
+
+  const [contractKits, psmKit, usdcIssuer, usdcBrand] = await Promise.all([
+    contractKitsP,
+    psmKitP,
+    E(agoricNames).lookup('issuer', 'USDC'),
+    E(agoricNames).lookup('brand', 'USDC'),
+  ]);
+
+  console.log('[CONTRACT_KITS]', contractKits);
+  console.log('[ISSUER]', usdcIssuer);
+
+  let govCreatorFacet;
+  for (const { psmGovernorCreatorFacet, label } of psmKit.values()) {
+    if (label === 'psm-IST-USDC') {
+      govCreatorFacet = psmGovernorCreatorFacet;
+      console.log('psm-IST-USDC found', label);
+    }
+  }
+
+  const psmPublicFacet = await E(govCreatorFacet).getPublicFacet();
+
+  let usdcMint;
+  for (const { publicFacet, creatorFacet: mint } of contractKits.values()) {
+    if (publicFacet === usdcIssuer) {
+      usdcMint = mint;
+      console.log('USDC found', mint);
+      break;
+    }
+  }
+
+  console.log('Minting USDC');
```

Comment:
This will break if it doesn't find a usdcMint. We should add an assertion so it'll fail deliberately if there are testnets where it's not found?